docs/configuration/settings.rst

```diff
@@ -5,113 +5,469 @@ Mautic Configuration Settings
+
+.. note:: 
+
+    Mailer is owner overrides any other name or Email to send mail from, including the default and individual Emails. Every Contact owner’s domain must have SPF and DKIM records. You can see this configuration for individual Emails, rather than globally.
+    For more information see :doc:`Mailer is owner</channels/emails>`
+
+* **Service to send mail through** - For most Mautic customers, this should remain as ``Mautic``. Your Emails route through the Mautic's Sparkpost account. If you are sending through your own Email service, select the service and enter your credentials. 
+  
 Default frequency rule
 ======================
 
+* **Do not contact more than <number> each <period>** - This limits the number of marketing messages a Contact receives in a certain period of time (day, week, month). Transactional messages don’t count towards this limit. You can adjust this at the individual Contact level, either manually or by preference center setting. 
+
+*More information is available in :doc:`Default Frequency Rule document</contacts/frequency_rules>`.*
+
 Monitored inbox settings
 ========================
 
+.. image:: images/monitored-settings.png
+  :width: 600
+  :alt: Screenshot showing Monitored Settings Configuration in Mautic
+
+* **Default Mailbox** - If Email addresses your messages are going to bounce, this inbox is where you receive those bounce notifications.
+
+* **Bounces** - A folder to monitor for new bounce messages or Emails.
+
+* **Unsubscribe Requests** - A folder to monitor for new unsubscribe requests. 
+
+* **Contact Replies** - Similar to the monitored inbox for bounces, this is the inbox Mautic checks for Contact replies. Using "Replies to Email" decisions in any Campaign requires configuration. With ``Use custom connection settings?`` set to ``no``, Mautic checks the default mailbox. If set to ``yes``, you may track a different mailbox for replies.
+
 Message settings
 ================
 
+.. image:: images/message-settings.png
+  :width: 600
+  :alt: Screenshot showing Message Settings Configuration in Mautic
+
+* **Text for the {webview_text} token** - The message indicating the reader can view the Email in their browser. The default is; 
+
+Having trouble reading this Email? Click here.
+To change the text, change the message between the ``<a href="|URL|">`` and ``</a>`` tags. Don't change the ``|URL|`` text, because that is a token for each Contact.
+
+* **Default Email signature** - The signature for your default Emails, which pairs with the name & Email address in the **Mail Send** settings.
+
+* **Append tracking pixel into Email body?** - To track Email opens, select **Yes**. Select **No** to prevent tracking, reporting on, and using decisions based on Email opens.
+
+* **Convert embed images to Base64** - Select **Yes** to display embedded images in Emails as code rather than as images.
```

Comment:
```suggestion
* **Convert embed images to Base64** - Select **Yes** to display embedded images in Emails using embedded base64 code rather than as embedded images.
```